lightning/src/onion_message/messenger.rs

```diff
@@ -270,6 +270,24 @@ pub trait MessageRouter {
 	fn find_path(
 		&self, sender: PublicKey, peers: Vec<PublicKey>, destination: Destination
 	) -> Result<OnionMessagePath, ()>;
+
+	/// Creates [`BlindedPath`]s to the `recipient` node. The nodes in `peers` are assumed to be
+	/// direct peers with the `recipient`.
+	///
+	/// The default implementation returns two-hop paths for each node in `peers` with the
+	/// `recipient` node (i.e., the peer is the introduction point).
+	fn create_blinded_paths<
+		ES: EntropySource + ?Sized, T: secp256k1::Signing + secp256k1::Verification
+	>(
+		&self, recipient: PublicKey, peers: Vec<PublicKey>, count: usize, entropy_source: &ES,
```

Comment:
`Router:: create_blinded_payment_paths` is used for `Invoice` paths, so presumably the implementation could decide, yeah.

`Message::create_blinded_paths` as of now just needs one as mentioned above. If we allow implementations to decide, are you saying if an implementation returns more than one then we should set all of them on an `Offer` or `Refund` (where QR code size may be a factor)? For reply paths, would we simply use an arbitrary (first?) one? Or even cycle through them in case of sending multiple `InvoiceRequest`s?